gapis/capture/decoder.go

```diff
@@ -89,10 +89,7 @@ func (d *decoder) endGroupImpl(ctx context.Context, id uint64, terminated bool)
 	case *cmdGroup:
 		obj.invoked = true
 		obj.cmd.SetTerminated(terminated)
-		id := d.builder.addCmd(ctx, obj.cmd)
-		for _, c := range obj.children {
-			c.SetCaller(id)
-		}
+		d.builder.addCmd(ctx, obj.cmd)
```

Comment:
Should we have a check here and fail the decode if there are children?

Reply:
Probably. I'll add it.